src/scene/renderer/forward-renderer.js

```diff
@@ -257,11 +257,13 @@ class ForwardRenderer extends Renderer {
                 this.shadowCascadeDistancesId[cnt].setValue(directional._shadowCascadeDistances);
                 this.shadowCascadeCountId[cnt].setValue(directional.numCascades);
                 this.lightShadowIntensity[cnt].setValue(directional.shadowIntensity);
-                this.lightSizeId[cnt].setValue(directional.lightSize);
+
+                const pixelsPerMeter = 1.0 / (lightRenderData.shadowCamera.renderTarget.width / directional.lightSize);
+                this.lightSizeId[cnt].setValue(pixelsPerMeter);
```

Comment:
I think to make it more understandable, lightSize uniform should be renamed to match its new meaning? `shadowPixelDensity` or something better?

Reply:
Fixed, named it shadowSearchArea.